experimental/transport-otlp-http/src/payload/OtelPayload.ts

```diff
@@ -52,22 +42,13 @@ export class OtelPayload {
         case TransportItemType.MEASUREMENT:
           const { toLogRecord, toResourceLog } = this.getLogTransforms;
 
-          const resourceLogWithMeta = this.resourceLogsWithMetas.find(({ resourceMeta }) =>
-            deepEqual(currentItemResourceMeta, resourceMeta)
-          );
-
-          if (resourceLogWithMeta) {
-            const { resourceLog } = resourceLogWithMeta;
-            // Currently the scope is fixed to '@grafana/faro-web-sdk'.
-            // Once we are able to drive the scope by instrumentation this will change and we need to align this function
-            resourceLog.scopeLogs[0]?.logRecords.push(toLogRecord(transportItem));
+          // Currently the scope is fixed to '@grafana/faro-web-sdk'.
+          // Once we are able to drive the scope by instrumentation this will change and we need to align this function
+          if (this.resourceLogs.length === 0) {
+            this.resourceLogs = [toResourceLog(transportItem)];
           } else {
-            this.resourceLogsWithMetas.push({
-              resourceLog: toResourceLog(transportItem),
-              resourceMeta: currentItemResourceMeta,
-            });
+            this.resourceLogs[0]?.scopeLogs[0]?.logRecords.push(toLogRecord(transportItem));
```

Comment:
Suggestions: add a short comment why we always can use the object at index zero.
"because teh faro batching takes care of grouping ... so we ... get either a single item or a list of items belonging to the same meta"